libs/prelude/Prelude/Types.idr

```diff
@@ -21,13 +21,12 @@ data Nat =
   | ||| Successor.
   S Nat
 
-%name Nat k, j, i
```

Comment:
Did you forget to put this back?

Reply:
This is unnecessary. The code can still type check.

Reply:
The `%name` pragma is not about type checking. Please don't remove this.